versioning/requirements.txt

```diff
@@ -1 +1,3 @@
+github_release
```

Comment:
After installing the requirements, I'm getting `ModuleNotFoundError: No module named 'github_release'`. Did you mean `githubrelease`?

Reply:
yes